src/containers/itemCollection/itemCollection.jsx

```diff
@@ -51,15 +51,58 @@ export class ItemCollection extends Component {
         })
     }
 
+    // This function splits a string by tabs/newlines and individually
+    // submits each item and then adds another edit item.
+    handleEditItemPaste = (pasteString) => {
+        //TODO does this work on Windows? Does it need to check for carriage return?
+        var splitStrings = pasteString.split(/[\t\n]/)
+
+        for (let itemName of splitStrings) {
+            //TODO Need to double check whether this logic of interacting with other functions is correct
+            const item = {
+                _id: uuid('item'),
+                name: itemName,
+                size: 1
+            }
+    
+            // Prevent the addition of an empty item, null item, or all whitespace item
+            if (item.name === null || item.name.match(/^\s*$/) !== null) {
+                continue;
+            }
+
+            const item1 = this.props.real.items.find(ele => ele._id === this.state._id)
+            const item2 = this.props.real.items.find(ele => ele.name === this.state.name)
+            // Check for duplicates. In this case, duplicates are not found, so add the item.
+            if (typeof item1 === 'undefined' && typeof item2 === 'undefined') {
```

Comment:
For duplicates, you can also now use the nosistack, to output a message explaining that there's a duplicate.

Reply:
See example in allArrangementsTable.jsx